src/app/pages/user-list/user-list.page.html

```diff
@@ -17,7 +17,7 @@
 
 
   <ion-list>
-    <ion-item button [routerLink]="['/user', user.id]" *ngFor="let user of (users | async)" lines="none">
+    <ion-item button [routerLink]="['/user', user.id]" *ngFor="let user of (users)" lines="none">
```

Comment:
Why did you remove 'async'? This might be the start of the issue behind the newly created user not showing up on the user-list page.